packages/server/test/node.spec.ts

```
@@ -21,7 +21,7 @@ describe('Node Specific Cases', () => {
       runTestsForEachServerImpl((testServer, serverImplName) => {
         // Deno and Bun does not empty responses
         if (!globalThis.Bun && !globalThis.Deno) {
-          it('should handle empty responses', async () => {
+          it.only('should handle empty responses', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)